rest/src/main/java/org/jboss/pnc/rest/restmodel/ProductVersionRest.java

```diff
@@ -158,6 +158,21 @@ public ProductVersion toProductVersion(Product product) {
 
     public ProductVersion toProductVersion(ProductVersion productVersion) {
         productVersion.setVersion(version);
+
+        if (currentProductMilestoneId != null) {
+            List<ProductMilestone> productMilestones = nullableStreamOf(productVersion.getProductMilestones())
+                    .filter(productMilestone -> productMilestone.getId().equals(currentProductMilestoneId))
+                    .collect(Collectors.toList());
+
+            if (!productMilestones.isEmpty()) {
+                productVersion.setCurrentProductMilestone(productMilestones.get(0));
+            } else {
+                productVersion.setCurrentProductMilestone(null);
+            }
+        } else {
```

Comment:
Is this else clause and the one nested above it necessary? It looks to me as if the current product milestone would already be null if they were invoked? Probably I don't understand the context of this, but just thought I'd check :)